src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -203,8 +203,11 @@
   </Target>
 
   <Target Name="CreatePerfTestListJson" DependsOnTargets="CreateAzureStorage" Condition="'$(Performance)' == 'true'">
-    <PropertyGroup>
-      <OtherRunnerScriptArgs>--perf-runner Microsoft.DotNet.xunit.performance.runner.Windows $(OtherRunnerScriptArgs)</OtherRunnerScriptArgs>
+    <PropertyGroup Condition="'$(TargetsWindows)' == 'true'">
+      <OtherRunnerScriptArgs>--perf-runner Microsoft.DotNet.xunit.performance.runner.Windows --osgroup windows $(OtherRunnerScriptArgs)</OtherRunnerScriptArgs>
+    </PropertyGroup>
```

Comment:
Since these properties are identical except for OSGroup, and there's literally a variable you could toLower called OSGroup, why not make these work?  (OSGroup would be Windows_NT).  Failing that, could still refactor this to using this logic to set $(OSGroupName) and set OtherRunnerScriptArgs once.